### DISCUSSION TITLE
Monero (XMR)

### DISCUSSION THREAD

**Discussion by trymeouteh** [MrNaif2018/bitcart#280] [Ideas]
Please consider adding support for Monero. Monero is the most popular privacy cryptocurrency and has been around for years.

**Comment by MrNaif2018** [comment]
Yeah, this is planned. But from my understanding neither the electrum nor the eth approach we have would work, so it would require developing an another (third) custom daemon implementation. It took a month of sponsored work for eth, so monero support would be delayed a bit till I can conduct another detailed research. One of the users also said that they might submit a PR for it, so will check that too

**Comment by trymeouteh** [comment]
Thank you for adding XMR!

**Comment by Jayd603** [comment]
Wait, XMR was added?   Excellent!
